tests/test_functions.py

```diff
@@ -15,9 +15,80 @@ def test_Constant():
     assert constant1.value == 0
 
     constant2 = Constant(5)
+    constant2.power = 2
```

Comment:
If the `__init__` in `Constant` has been updated then you can use
```suggestion
    constant1 = Constant(5, 2)
```
Also when using a new assert statement you can reuse the variable names.